src/libproxychains.c

```diff
@@ -86,13 +87,13 @@ static int init_l = 0;
 
 static void get_chain_data(proxy_data * pd, unsigned int *proxy_count, chain_type * ct);
 
-static void* load_sym(char* symname, void* proxyfunc) {
+static void* load_sym(char* symname, void* proxyfunc, int is_mandatory) {
 
 	void *funcptr = dlsym(RTLD_NEXT, symname);
 
 	if(!funcptr) {
-		fprintf(stderr, "Cannot load symbol '%s' %s\n", symname, dlerror());
-		exit(1);
+		proxychains_write_log(LOG_PREFIX "Cannot load symbol '%s', is_mandatory=%d, %s\n", symname, is_mandatory, dlerror());
```

Comment:
it's not necessary to change log function, iirc at this point fprintf is used because proxychains isnt fully set up yet.

Reply:
i think if not mandatory we best dont log this at all. otherwise users without close_range will get spam whenever they use proxychains.

Reply:
fixed in latest commit. 
I have reverted to the old `fprintf`, and the error only shows for mandatory symbol.